xinference/api/restful_api.py

```diff
@@ -1554,10 +1554,14 @@ async def create_chat_completion(self, request: Request) -> Response:
         if body.tools and body.stream:
             is_vllm = await model.is_vllm_backend()
 
-            if not is_vllm or model_family not in QWEN_TOOL_CALL_FAMILY:
+            if not (
```

Comment:
Shall us enable fake stream for those models that does not support stream tool calls?